content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -182,7 +182,10 @@ If you do not want to have (major) version numbers increase quickly, like with f
      <jenkins.version>2.176.4</jenkins.version>
 ----
 +
-Here the version numbers will look like `1.321.vabcdef456789` or `1.999999-SNAPSHOT`, respectively, which are more similar to version numbers used by Jenkins itself.
+Here the version numbers will look like `1.321.vabcdef456789` or `1.999999-SNAPSHOT`, respectively.
+This could be appropriate if you are leery of committing up front to having major version numbers be in the triple digits,
+with no option of going back to `maven-release-plugin`-style versioning except by starting at say `1000.1`,
+because version numbers going forward must be mathematically larger than any currently on the update center.
```

Comment:
Prefer sentence-per-line formatting here.

Reply:
I am deliberately using a line per major phrase, as these are most likely to be units of editing.